geowebcache/core/src/main/java/org/geowebcache/config/TileLayerConfiguration.java

```diff
@@ -155,17 +155,6 @@
      */
     boolean containsLayer(String layerName);
 
-    /**
-     * If this method returns TRUE WMTS service implementation will strictly comply with the
-     * correspondent CITE tests.
-     *
-     * @return TRUE or FALSE, activating or deactivation CITE strict compliance mode for WMTS
-     */
-    default boolean isWmtsCiteCompliant() {
-        // by default CITE tests strict compliance is not activated
-        return false;
-    }
-
     /**
```

Comment:
I take it this isn't necessary int the TileLayerConfiguration, since it's now taken care of in ServerConfiguration?